main/EDMainMod.F90

```diff
@@ -159,6 +161,10 @@ subroutine ed_ecosystem_dynamics(currentSite, bc_in, bc_out)
        call currentSite%flux_diags(el)%ZeroFluxDiags()
     end do
 
+
```

Comment:
Short comment describing that this is zero'ing history diagnostic variables on fates columns (sites)

Reply:
Yes can do